lib/fluent/plugin/kubernetes_metadata_common.rb

```diff
@@ -86,7 +86,8 @@ def parse_pod_metadata(pod_object)
           'pod_id'         => pod_object['metadata']['uid'],
           'pod_name'       => pod_object['metadata']['name'],
           'containers'     => syms_to_strs(container_meta),
-          'host'           => pod_object['spec']['nodeName']
+          'host'           => pod_object['spec']['nodeName'],
+          'pod_ip'         => pod_object['status']['podIP']
```

Comment:
Any chance status is nil?  otherwise lgtm

Reply:
After looking into the Kubernetes docs, it seems that pods must have a status.

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#pod-v1-core

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status

This particular paragraph is the most interesting:
"All objects that represent a physical resource whose state may vary from the user's desired intent SHOULD have a "spec" and a "status". Objects whose state cannot vary from the user's desired intent MAY have only "spec", and MAY rename "spec" to a more appropriate name."